content/en/docs/how-tos/onboarding-a-new-component.md

```diff
@@ -241,21 +241,85 @@ jobs BEFORE you reference them from another component (via the image-references
+
+**Steps**:
+1. If a name change was not directly requested by a staff engineer, ensure that a staff engineer agrees on the name change (`@aos-staff-engineers` on Slack).
+2. Open and `/hold` a pull request, PR1, against github.com/openshift/release to change the operand component's name in CI. In virtually all cases, this will be a PR against the component's ci-config for its main branch. A component's name should be specified in the [`images.to:`](https://github.com/openshift/release/blob/784b1d65e985e19b94fd65bb26c6c5a4fecc2ea8/ci-operator/config/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-master.yaml#L37-L47) stanza of a component's CI configuration. The new tag name should match exactly what will appear in `oc adm release info` when the component is listed.  
+3. Open and `/hold` a pull request, PR2, against the repo(s) of the operator(s) which references the old operand component name in its `image-references` file. PR2 should update `image-references` to use the new component name.
+4. Open and `/hold` a pull request, PR3, against github.com/openshift/ocp-build-data in the openshift-4.x branch of the targeted release(s). Note that ART may have already cut a new release branch, meaning you need to open a PR for openshift-4.x+1 as well. The PR should change the `name` field in the component's metadata. Note that in ART metadata, the desired name should be prefixed with `ose-`. For example, the `etcd` operand component is name definition [can be seen here, for openshift-4.11](https://github.com/openshift/ocp-build-data/blob/129ef9fb8335991d88ce9f41a61057c6ab2bb08e/images/cluster-etcd-operator.yml#L23).  
+5. Submit a copy of [this ART team template](https://issues.redhat.com/browse/ART-1443) to communicate to ART that a component name change is desired. Include the PRs in the Jira ticket.
+6. PR1 and PR3 should be passing tests and ready to merge (PR2 will be failing at this point). No PRs other than PR1 and PR2 should be merged in their respective github.com repositories during the following process. In a synchronous chat with an ART team member or release manager (`@release-artists` in `#aos-art` on Slack) the following should be performed:
+   1. On the central app.ci CI cluster, a release-artist should check the current registry.ci.openshift.org image associated with the component in the `-n ocp is/4.{minor}` release image stream. For example, `oc -n ocp get istag 4.11:{old-component-name} -o=json | jq .tag.from.name` should output a pullspec like `registry.ci.openshift.org/ocp/4.11@sha256:...` for 4.11.
+   2. The release-artist should run `oc -n ocp tag {existing-registry.ci-openshift-org@sha26..} 4.{minor}:{new-component-name}` to establish a tag with the new operand component name.
+   3. Run `/retest` on PR2(s). Tests should now pass.
+   4. Unhold and merge PR1.
+   5. Unhold and merge PR2(s).
+   6. Unhold and have the release-artist merge PR3.
```

Comment:
we should note here that in the gap between PR2 and PR3 merging (which could be extensive), ART automation will likely rebuild the operator with the wrong reference and we'll get a failed nightly.

we can avoid this if someone can green-button the PR2 merge and PR3 immediately after, but it seems likely to happen if we wait for the typical automated merge process, so it seems worthy of mention.

Reply:
I ask for a synchronous chat for these flows to minimize windows like this. I don't talk about all the poor outcomes of not performing these steps in quick succession, but I'll add a note on this one.